react-validated-input-component/src/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCheck, faTimes } from '@fortawesome/free-solid-svg-icons';
+
+const MIN_PASS_LEN = 8;
+
+class PasswordInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isValid: false,
+      errorMsg: 'A password is required.',
+      password: ''
+    };
+  }
+
+  handleChange = e => {
+    this.setState({
+      password: e.target.value
+    });
+  }
+
+  handleSubmit = e => {
+    e.preventDefault();
+    if (this.state.password.length === 0) {
+      this.setState({ errorMsg: '' });
```

Comment:
Here the error message should be something like: `"Password is required"`

Reply:
![react-validated-input2](https://user-images.githubusercontent.com/82009814/125367171-871fb380-e32c-11eb-9684-d03db291426a.gif)